pkg/controllers/machinesync/machine_sync_controller_test.go

```
@@ -1439,92 +1439,95 @@ var _ = Describe("With a running MachineSync Reconciler", func() {
-					capaMachine.SetOwnerReferences([]metav1.OwnerReference{
-						{
-							Kind:               machineKind,
-							APIVersion:         clusterv1.GroupVersion.String(),
-							Name:               testMachine.Name,
-							UID:                testMachine.UID,
-							BlockOwnerDeletion: ptr.To(true),
-							Controller:         ptr.To(false),
-						},
-					})
-				})).Should(Succeed())
+				By("Creating a sentinel CAPI machine", func() {
```

golangci-lint (2.12.2)

[error] missing whitespace above this line (no shared variables above expr)

(wsl_v5)